exampleSite/utilities/README.md

```diff
@@ -42,11 +42,18 @@ Guidelines regarding sponsor logo files and formatting:
 
 * The dimensions of the image file must be 200px square.
 * The background must be either white or transparent.
-* There must *not* be a border (one will be added using CSS).
 
-All logos will be constrained, via markup, to 100px square; combined with the image file dimensions, this allows for high-density display (ex. Retina) compatibility.
+All logos will be resized at release time, to 200px wide. Versions for high-density display (ex. Retina) will also be created. It is recommended that you do not use a sponsor logo that is smaller than 200px wide to keep from quality degradation at resize time.
 
-Use [contrib/resize_sponsor_logo.sh](resize_sponsor_logo.sh) to convert a logo centered into the right dimensions.
+Use [contrib/resize_sponsor_logo.sh](resize_sponsor_logo.sh) to convert a logo centered into the right dimensions. *Note - this script is no longer needed in the new theme*
+
+## Team members
+
+See the example team members listed in the generated data file. You can now add additional fields for each team member, as well as a photo. The photo must be in JPG format, and should be 300px x 300px. These images are located in the `static/events/YYYY-CITY/organizers` directory.
+
+## Social sharing image
+
+A sharing image is added to the Open Graph tags for your event pages, to improve the sharing on social networks such as Facebook (or in Slack). This image must be named `sharing.jpg` and located in `static/events/YYYY-CITY`. It should be a minimum 1200px x 630px, and use ratio: 1.91:1.
```

Comment:
If they don't add one, then there is none, which means no image is pulled by Facebook. But it bears clarification.